README.md

```diff
@@ -7,8 +7,8 @@
 3. Execute os comandos que seguem:
     ```bash
     mkdir volumes
-    git clone https://github.com/LibreCodeCoop/mediawiki-development/ volumes/mediakiwi
-    git clone git@github.com:LibreCodeCoop/mapa-colaborativo.git volumes/src
+    git clone https://github.com/LibreCodeCoop/mediawiki-development/ volumes/mediawiki
+    git clone https://github.com/LibreCodeCoop/mapa-colaborativo.git volumes/src
```

Comment:
Fix ok: ~🥝~wiki

revert this:
```suggestion
    git clone git@github.com:LibreCodeCoop/mapa-colaborativo.git volumes/src
```
When we clone as https, will be necessary use the password every when we need to make a push